### PR TITLE
[Task #48513] UserAdmin

### PR DESCRIPTION
**Pull request by giangnguyen20**
## Related Tickets:
- [#48513](https://edu-redmine.sun-asterisk.vn/issues/48513)

## WHAT this PR does?
- Taov view quản lý người dùng.
- Xử lý logic cho active hoặc block người dùng.

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by giangnguyen20** [comment]
ready

**Review by manhtd-1083 — COMMENTED** [review]
conflicted

**Comment by giangnguyen20** [comment]
ready

**Review by manhtd-1083 — COMMENTED** [review]
conflicted

**Comment by giangnguyen20** [comment]
ready

File: resources/views/admin/users/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            <th scope="col">{{ __('Name') }}</th>
+            <th scope="col">{{ __('Email') }}</th>
+            <th scope="col">{{ __('Phone') }}</th>
+            <th scope="col">{{ __('Status') }}</th>
+            <th scope="col">{{ __('Role') }}</th>
+            <th scope="col">{{ __('Edit') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach($users as $key => $item)
```
Comment (manhtd-1083) [inline resources/views/admin/users/index.blade.php:28]:
```suggestion
        @foreach ($users as $key => $item)
```

**Comment by giangnguyen20** [comment]
ready

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -70,4 +75,4 @@
         </div>
     </div>
 </div>
-@endsection
+@endsection
```
Comment (manhtd-1083) [inline resources/views/auth/login.blade.php:78]: eof

File: app/Http/Controllers/Auth/LoginController.php [hunk 3]
```diff
@@ -44,9 +45,33 @@ public function __construct()
...
+        if ($compare) {
+            if (Auth::user()->role_id == config('auth.roles.admin')
+                && Auth::user()->status == config('auth.status.active')) {
+                return redirect()->route('admin.index');
+            } else {
+                if (Auth::user()->status == config('auth.status.active')) {
+                    return redirect()->route('home');
+                } else {
+                    Auth::logout();
+                    return redirect()->route('login')->with('messages', 'user_lock');
```
Comment (manhtd-1083) [inline app/Http/Controllers/Auth/LoginController.php:70]:
```suggestion
                    Auth::logout();

                    return redirect()->route('login')->with('messages', 'user_lock');
```

**Review by manhtd-1083 — COMMENTED** [review]
conflicted

**Comment by giangnguyen20** [comment]
ready

**Pull request merged**